be/src/vec/exec/jni_connector.h

```diff
@@ -205,6 +216,17 @@ class JniConnector {
      */
     std::map<std::string, std::string> get_statistics(JNIEnv* env);
 
+    /**
+     * Call java side function JniScanner.getTableSchema.
+     *
+     * The schema information are stored as a string.
+     * Use # between column names and column types.
+     *
+     * like: col_name1,col_name2,col_name3#col_type1,col_type2.col_type3
+     *
+     */
+    Status get_table_schema(std::string& table_schema_str);
```

Comment:
Map<string, string> get_table_schema();
`field_name` -> `array<struct<col1: int, col2: string>>`

Reply:
Here I use the nested format of json.
You can see `TableSchema.SchemaColumn`